assets/javascripts/discourse/lib/ai-bot-helper.js

```diff
@@ -0,0 +1,27 @@
+import { ajax } from "discourse/lib/ajax";
+import Composer from "discourse/models/composer";
+import I18n from "I18n";
+
+export async function ComposeAiBotMessage(targetBot, composer, appEvents) {
```

Comment:
Up to you, but we generally don't use this pattern of capitalized names for single usage function, and would more likely go with: `composeAiBot
Message`